### PR TITLE
DB-426: Disable addon management pages

### PR DESCRIPTION
**Pull request by maxim-cliqz**

### CODE REVIEW

**Comment by alver-cliqz** [comment]
retest this please

**Comment by luciancor** [comment]
retest this please

File: mozilla-release/toolkit/mozapps/extensions/content/extensions.js [hunk 1]
```diff
@@ -1746,9 +1746,19 @@ var gCategories = {
     this.node = document.getElementById("categories");
     this._search = this.get("addons://search/");
 
+    // These addon categories are disabled in CLIQZ.
+    const disabledCategories = new Set([
```
Comment (luciancor) [inline mozilla-release/toolkit/mozapps/extensions/content/extensions.js]: @maxim-cliqz can you please use "extensions.cliqz.listed" pref and only disable the extension page if that pref is false? Just like in  https://github.com/cliqz-oss/browser-f/blob/master/cliqz.cfg#L312

Comment (maxim-cliqz) [inline mozilla-release/toolkit/mozapps/extensions/content/extensions.js]: @luciancor done

**Pull request merged**